pkg/kubelet/kubelet.go

```diff
@@ -401,8 +402,17 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 			options.FieldSelector = fields.Set{metav1.ObjectNameField: string(nodeName)}.String()
 		}))
 		nodeLister = kubeInformers.Core().V1().Nodes().Lister()
+		nodeInformer := kubeInformers.Core().V1().Nodes().Informer()
 		nodeHasSynced = func() bool {
-			return kubeInformers.Core().V1().Nodes().Informer().HasSynced()
+			return nodeInformer.HasSynced()
+		}
+		_, err := nodeInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
I think it does not call addfunc for existing objects. Since it does not call for existing objects, I feel that there is no need to add an event handler. WDYT?